Automattic/wp-calypso
# Jetpack Cloud: add error message when scan fails and threats were found

#### Changes proposed in this Pull Request

* Display an error message for the case in which the Scan failed, while at the same time, it found threats.

#### Implementation notes

* I noticed that in a previous PR I created a file named `useThreats.ts`. This doesn't follow our conventions, so I renamed it to `use-threats.ts` and updated its imports.

#### Testing instructions

* Go to JP Cloud
* Select a site that has Scan enabled and has threats
* On devtools, go to the components tabs, find the ScanThreats component, and set the `error` prop to true 
![image](https://user-images.githubusercontent.com/3418513/81103312-3bcb7300-8ee7-11ea-94ba-90c113ace232.png)
* Verify the error message appears below the list of threats
* Verify the link to support works
* Verify the `scan again` link/button triggers a new scan run

1151678672052943-1174314790666757

#### Demo

Scan on-demand **enabled**
![image](https://user-images.githubusercontent.com/3418513/81103519-851bc280-8ee7-11ea-9183-0ca5ceba8eb7.png)

Scan on-demand **disabled**
![image](https://user-images.githubusercontent.com/3418513/81184125-6587a700-8f86-11ea-977d-6fb0340fc59e.png)
